recipe_scrapers/donnahay.py

```diff
@@ -8,23 +8,23 @@ class DonnaHay(AbstractScraper):
     def host(cls):
         return "donnahay.com.au"
 
-    def author(self):
-        return self.schema.author()
-
     def title(self):
-        return self.schema.title()
-
-    def category(self):
-        return self.schema.category()
-
-    def total_time(self):
-        return self.schema.total_time()
+        return (
+            self.soup.find("h1", class_="text-center recipe-title__mobile")
+            .getText()
+            .upper()
+        )
 
     def yields(self):
-        return self.schema.yields()
+        div = self.soup.find("div", class_="col-sm-6 method")
+        return div.find("b").getText()
```

Comment:
Some of the older recipes such as https://www.donnahay.com.au/recipes/lunch/asian-style-prawn-and-rice-omelette don't have the serves in a separate \<b\> tag. Might need to get this detail another way.